lib/apress/selenium_eti/spec/company_site/mini_eti/minieti_spec.rb

```diff
@@ -17,108 +17,126 @@
+
+      it 'счетчик изображений увеличится на 1' do
+        expect(@cs_eti_table_products.image(@product)[:counts]).to eq(@image[:counts] + 1)
+      end
 
       it 'увеличивается заряд батарейки' do
         expect(@cs_eti_table_products.battery(@product)[:level])
-          .to eq(@battery[:level] + CONFIG['battery_percents']['price'])
+          .to eq(@battery[:level] + CONFIG['battery_percents']['image'])
       end
     end
 
-    context 'когда заполняем цену от и до' do
-      before(:all) do
-        @name = Faker::Number.number(5)
-        @cs_eti_table_products.add_product(name: @name)
-        @product = @cs_eti_table_products.product(name: @name)
-        @battery = @cs_eti_table_products.battery(@product)
-        @price_range = {type: :range, price: Faker::Number.number(2), price_max: Faker::Number.number(3)}
+    describe 'Установка розничных цен', feature: 'company_site/eti/product_price_spec: Установка цен' do
+      context 'когда цена точная' do
+        before(:all) do
+          @name = Faker::Number.number(digits: 5).to_s
+          @price = {type: :exact, price: Faker::Number.number(digits: 3)}
+          @cs_eti_table_products.add_product(name: @name)
+          @product = @cs_eti_table_products.product(name: @name)
+          @battery = @cs_eti_table_products.battery(@product)
+          @cs_eti_table_products.set_price(@product, @price)
+        end
 
-        @cs_eti_table_products.set_price(@product, @price_range)
-      end
+        it 'введенная цена отображается' do
+          expect(@cs_eti_table_products.price(@product)).to include(@price[:price].to_s)
+        end
 
-      it 'введенная цена отображается' do
-        expect(@cs_eti_table_products.price(@product)).to include @price_range[:price], @price_range[:price_max]
+        it 'увеличивается заряд батарейки' do
+          expect(@cs_eti_table_products.battery(@product)[:level])
+            .to eq(@battery[:level] + CONFIG['battery_percents']['price'])
+        end
       end
-    end
 
-    context 'когда заполняем цену со скидкой' do
-      before(:all) do
-        @name = Faker::Number.number(5)
-        @cs_eti_table_products.add_product(name: @name)
-        @product = @cs_eti_table_products.product(name: @name)
-        @battery = @cs_eti_table_products.battery(@product)
-        @discount_price = {
-          type: :discount,
-          price: Faker::Number.number(3),
-          new_price: Faker::Number.number(2),
-          expires_at: Time.now.strftime('%d.%m.%Y'),
-        }
+      context 'когда цена от и до' do
+        before(:all) do
+          @name = Faker::Number.number(digits: 5).to_s
+          @cs_eti_table_products.add_product(name: @name)
+          @product = @cs_eti_table_products.product(name: @name)
+          @battery = @cs_eti_table_products.battery(@product)
+          @price_range = {
+            type: :range, 
+            price: Faker::Number.number(digits: 2), 
+            price_max: Faker::Number.number(digits: 3)
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.